src/types.ts

```diff
@@ -64,7 +64,7 @@ export interface IBadge {
 export type Transaction = {
   type: string;
   id: string;
-  title: string;
+  title: any;
```

Comment:
```suggestion
  title: string | React.ReactNode;
```

@secondl1ght can you try this one?